v2Tov1Migrations/migrate_regions.js

```diff
@@ -0,0 +1,61 @@
+require('dotenv').config();
+const ProgressBar = require('progress');
+const { Writable } = require('stream');
+
+const ws = Writable({ objectMode: true });
+const { knex } = require('../database/knex');
+
+async function migrate() {
+  const base_query_string = `SELECT pr.* FROM public.region pr left join regions.region rr on pr.region_uuid = rr.id 
+  where rr.id is null and pr.type_id = 2`;
```

Comment:
@Kpoke can you explain this?

Reply:
The regions table is so large about 6 million entries and I am not sure all of them are being used, so my idea for this is to migrate this based on the types that are needed, like freetown, then grids etc.